### PR TITLE
phpMyAdmin: update to 5.2.0, fix update check.

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by TinfoilSubmarine** [comment]
I just saw the note in https://github.com/ampache/ampache/wiki/Installation:
> PHP = 7.4|8.0|8.1 (PHP 8.1 support is **unstable**)

I guess I will bump this down to `php8.0`.

File: srcpkgs/phpMyAdmin/update [hunk 1]
```diff
@@ -0,0 +1,2 @@
+site="https://www.phpmyadmin.net/files/"
+pattern="<a href=.*>\K[\d.]+(?=</a>)"
```
Comment (williamdes) [inline srcpkgs/phpMyAdmin/update:2]: You should change this to https://www.phpmyadmin.net/home_page/version.txt
See: https://github.com/linuxserver/docker-phpmyadmin/issues/10

**Review by williamdes — APPROVED** [review]

**Comment by Francesco149** [comment]
ok to adopt the package btw, if anyone wants

**Review by williamdes — APPROVED** [review]
Thank you for for taking maintainership

**Comment by williamdes** [comment]
Please always mention me on phpmyadmin PRs around the web

**Pull request merged**